proxy/src/coco/source.rs

```diff
@@ -188,6 +188,10 @@ pub struct TreeEntry {
     pub path: String,
 }
 
+/// Representation of high-level project [`Stats`] - its known number of branches,
+/// commits, and contributors on the default branch
+pub struct Stats(pub surf::vcs::git::Stats);
```

Comment:
Something I realised is that since this is just for the `ToDocumentedType` nonsense, we could define it in `http/source.rs` like:
```rust
struct DocumentStats;

impl ToDocumentType DocumentStats { /* etc */ }
```

Because really it has nothing to do with `surf::Stats`

Reply:
Done ✨